tests/api2/test_system_lifetime.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+from middlewared.test.integration.utils import call
+
+
+def test_system_reboot():
+    boot_id = call("system.boot_id")
+
+    call("system.reboot", "Integration test")
+
+    for i in range(180):
+        try:
+            new_boot_id = call("system.boot_id")
+        except Exception:
+            pass
+        else:
+            if new_boot_id == boot_id:
```

Comment:
How does this work? `new_boot_id` will never match the `boot_id` since the system will have been rebooted?

Reply:
Yes, this was a mistake. The test was not testing the fact that the system was indeed rebooted. Updated.

Reply:
http://jenkins.eng.ixsystems.net:8080/job/tests/job/api_tests/842/